airflow/www/static/js/tree/dagRuns/index.test.jsx

```diff
@@ -42,13 +42,13 @@ const Wrapper = ({ children }) => {
           <ContainerRefProvider value={{}}>
             <TimezoneProvider value={{ timezone: 'UTC' }}>
               <AutoRefreshProvider value={{ isRefreshOn: false, stopRefresh: () => {} }}>
-                <SelectionProvider value={{ onSelect: () => {}, selected: {} }}>
+                <MemoryRouter>
```

Comment:
Why do we use MemoryRouter here but BrowserRouter elsewhere?

Does MemoryRouter mean that components here don't change the URL, right?

Reply:
Oh. This is a `test.jsx` :D